### DISCUSSION TITLE
Update Default SVG icons for more consistent sizing.

### DISCUSSION THREAD

**Discussion by cjpillsbury** [muxinc/media-chrome#139] [Proposals]
**NOTE**: While this does not have to be resolved immediately, it may impact decisions and implementation details for [Open Elements](https://github.com/muxinc/elements) Mux Player goals.

### Overview

There are a number of inconsistencies in our current default SVG icons for media-chrome which lead to:
- surprising transparent clickable regions and sizing/layout requirements when using icons in `"center-chrome"` examples (e.g. big button/mobile use cases)
- height/width changes for components with multiple icons (play vs. pause icons, different volume icons)
- general inconsistencies between icon DIMs

Our team's designer has provided a set of icons that we're intending on using internally for Mux Player (bottom row) that are slightly stylized. He's also graciously provided some "more standard but cleaned up" svgs that account for the above. Both of these resolve the above concerns. Ideally, one of these two sets are an acceptable alternative to the current veresions, and we can knock out a quick PR that swaps them out (regardless of which is considered). However, I thought there should be some review/approval before taking on this effort.

<img width="938" alt="Screen Shot 2022-01-26 at 11 59 56 AM" src="https://user-images.githubusercontent.com/2389566/151255756-c2e23ec8-b77f-482e-b2e2-ae29a14719d7.png">

**Comment by cjpillsbury** [comment]
To reiterate/rephrase: This is could be de-escalated upon discussion/review, but some initial evaluation should be prioritized, even if "we should defer on this for now in Media Chrome" is the immediate decision.

**Reply by heff** [reply]
My S1 is "go for it" and I like the 2nd and 3rd options. @mmcc sometimes has icon prefs.

**Reply by mmcc** [reply]
I think I lean towards number 2. There are some nice features of 3, but I'm not a huge fan of the sound icons, and just generally I worry that the really thin weight is going to cause issues as a generalist player iconset.

**Reply by cjpillsbury** [reply]
> I think I lean towards number 2. There are some nice features of 3, but I'm not a huge fan of the sound icons, and just generally I worry that the really thin weight is going to cause issues as a generalist player iconset.

I'm going to treat this as answered via ☝️

**Comment by luwes** [comment]
as long as we don't add a gear icon I like it!  
nice improvement of the PiP icon in the last row and love the CC icon!

**Reply by heff** [reply]
I'm very interested to hear more on this gear icon hatred. Not that I have any opinion on it.

**Reply by luwes** [reply]
it's not my original opinion actually but someone once made me see it's quite an ugly and out of place icon to represent preferences or settings and especially for a video player. I think at the time we compared it to physical media devices like amps, tuners, etc where you have all these neat dials, knobs, sliders, toggles (you're not touching gears right :)
eventually the gear icon was still chosen for the video player 😞. but I'm hopeful they'll be gone soon 😃

**Reply by cjpillsbury** [reply]
that resonates w/me.

**Reply by heff** [reply]
I get that. It's a little like the "save" icon still being a 3.5" floppy disk.

What's the alternative though? How do you represent preferences otherwise, or "a lot of neat dials in one place"? Or do you just fit a lot more icons onto the control bar?

**Reply by heff** [reply]
I might have found an answer to my own question.
![image](https://user-images.githubusercontent.com/166/151461054-b71fb87d-2847-4d45-87a1-ff8c3850955a.png)
I do like that better.

**Reply by gkatsev** [reply]
Hamburger menu? Or maybe bento menu to imply there might be submenus?
![](https://preview.redd.it/fjl4a84ouro01.png?auto=webp&s=097525911e00a804efe359f900e303598a7416ef)

**Reply by heff** [reply]
Those are good too. I forget which player, but one of streaming platforms crams more controls into the ~~vertical triple dot menu~~ kebab menu as the player gets smaller. Maybe that's a good way to think about it. "Everything we couldn't fit onto the control bar."

**Reply by luwes** [reply]
Added some that I saved before in this Figma file.
https://www.figma.com/file/JMtp0bKNoXZyZKFMP2YFm3/Preference-Icons?node-id=0%3A1

That's Wistia, if you resize here you can see https://dev.playerx.io/players/wistia/
I like that approach very much actually, similar to how the Apple touch bar works.
Then there's no need for a menu panel that slides open on click of a menu title.
https://support.apple.com/guide/macbook-pro/13-inch-macbook-pro-use-the-touch-bar-apdf98ce3d2b/mac

**Reply by cjpillsbury** [reply]
Chrome uses kebab as well
<img width="800" alt="Screen Shot 2022-01-28 at 7 22 19 AM" src="https://user-images.githubusercontent.com/2389566/151555298-a60f8820-3a8a-4a98-83d0-33ae2b5f55ac.png">

**Comment by cjpillsbury** [comment]
Since the rest of this discussion is extending beyond the scope of currently defined/needed icons. I'm going to mark this as answered.
